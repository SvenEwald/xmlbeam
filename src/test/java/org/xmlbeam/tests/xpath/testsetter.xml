<a>
	<b>
		<c>Content</c>
	</b>
	<d/>
</a>