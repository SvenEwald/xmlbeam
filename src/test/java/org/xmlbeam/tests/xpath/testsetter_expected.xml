<a>
	<b>
		<c>Content</c>
	</b>
	<d><c>Content</c></d>
</a>